### ISSUE TITLE
Wrong accuracy was chosen as model_best.mlmodel

### ISSUE THREAD

**Issue by MrOCR** [mittagessen/kraken#96]
1) It should have chosen `model_5.mlmodel` as `best` because it had higher accuracy.
2) Also, there is wrong spelling in the last line it said `model_0.mlmdel`, shouldn't it supposed to be `model_0.mlmodel`? 
https://github.com/mittagessen/kraken/search?q=mlmdel&unscoped_q=mlmdel
```
(kraken) home@home-lnx:~/Desktop/kraken-master$ ketos train ./training_data/*.png
Building training set  [####################################]  100%          
[2.3791] alphabet mismatch {'ـ'} 
Initializing model ✓
epoch 0/∞  [####################################]  13506/13506          
Accuracy report (0) 0.9931 9342 64
epoch 1/∞  [####################################]  13506/13506          
Accuracy report (1) 0.9957 9342 40
epoch 2/∞  [####################################]  13506/13506          
Accuracy report (2) 0.9972 9342 26
epoch 3/∞  [####################################]  13506/13506          
Accuracy report (3) 0.9972 9342 26
epoch 4/∞  [####################################]  13506/13506          
Accuracy report (4) 0.9976 9342 22
epoch 5/∞  [####################################]  13506/13506          
Accuracy report (5) 0.9978 9342 21
Moving best model model_0.mlmdel (0.9931492185827446) to model_best.mlmodel
```

**Comment by mittagessen** [comment]
Early stopping can be implemented that way (taking the absolute best or last model of the run) but we didn't mainly for overfitting reasons. The new best model is the one that shows an improvement on the validation set above the significance threshold (default: 0.005) as any improvement below that is deemed to be noise.

As your accuracy on the evaluation set is extremely high (I assume you're training on artificial data?) the default threshold is too high as the model is still improving when the scheduler aborts the training loop. Set `--min-delta` to something like `5e-4` or even lower will train to full convergence.

**Comment by MrOCR** [comment]
It's true, I was training on synthetic images. The terabyte model is still a dream to me.
Thanks for the tweaks.
